### PR TITLE
Align autocomplete on scroll

### PR DESCRIPTION
**Pull request by ThoWagen**
This PR adds a event listener that aligns the autocomplete on scroll.

This is needed for the PR in VuFind https://github.com/vufind-org/vufind/pull/3226

### CODE REVIEW

**Comment by demiankatz** [comment]
@crhallberg, can you take a look at this and issue a new release if you're happy with it?

File: autocomplete.js [hunk 1]
```diff
@@ -276,6 +276,16 @@ function Autocomplete(_settings) {
           },
           false
         );
+        window.addEventListener(
+          "scroll",
+          function acresize() {
```
Comment (demiankatz) [inline autocomplete.js]: Would it make sense to share a function between the resize and scroll event listeners instead of copying and pasting the code?

**Review by demiankatz — COMMENTED** [review]
One small question based on something I just noticed -- we have two identical event handlers defined here. Maybe it's best to keep them separate for future development, but just leaving a comment to consider the alternative....

File: autocomplete.js (hunk 1 above)
Comment (crhallberg) [reply autocomplete.js]: I would agree with this. Can you put the function above the event listeners and reference it?

Comment (ThoWagen) [reply autocomplete.js]: Shouldn't it be possible to move 
```
if (lastInput === false) {
    return;
}
```
inside of `align`. Then we don't need another method for this. (`() => _align(lastInput)` does the trick then)

Comment (demiankatz) [reply autocomplete.js]: That makes sense to me -- we'd just have to move the declaration of lastInput a little higher in the code. @crhallberg, do you have any concerns about this?

Comment (ThoWagen) [reply autocomplete.js]: > we'd just have to move the declaration of lastInput a little higher in the code

That's not necessary if we use 
```
if (input === false) {
    return;
}
```
instead. Committed the simplified version.

**Review by demiankatz — APPROVED** [review]
This optimization makes sense to me, and the code looks good. However, I'll leave this for @crhallberg to merge, in case he has any remaining concerns.

**Pull request merged**